### ISSUE TITLE
After the latest version of obsidian, the table cannot be previewed and automatically adjusted in real time.

### ISSUE THREAD

**Issue by seven-Jack** [tgrosinger/advanced-tables-obsidian#260]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - Obsidian Version: [e.g. v0.9.10] (Settings → About → Current Version)
 - Advanced Tables Version: [e.g. 0.4.0] (Settings → Third-party plugin → Browse → Advanced Tables → currently installed)

**Additional context**

**Comment by zwa-vault** [comment]
Confirmed, same issue after updating to the lastest verstion just now!
Version v1.3.4
Installer version v.0.15.8

**Comment by bhuveh** [comment]
Yeah all the formatting in source mode and live mode is gone after the updates. :(

**Comment by zwa-vault** [comment]
@bhuveh True, a new line above the table solves the problem. But still it's a bug. A new line should not be required!

**Comment by tgrosinger** [comment]
Sounds like this is an issue in Obsidian. Please see [this forum post](https://forum.obsidian.md/t/tables-dont-render-in-live-preview-unless-theres-a-blank-line-above/60239)
